functable.h

```diff
@@ -36,6 +36,26 @@ struct functable_s {
     uint32_t (* update_hash)        (deflate_state *const s, uint32_t h, uint32_t val);
 };
 
-Z_INTERNAL extern Z_TLS struct functable_s functable;
+/* Platform and compiler has pointer size atomic store */
+#if defined(X86_FEATURES) || defined(ARM_FEATURES) \
+    || defined(PPC_FEATURES) || defined(POWER_FEATURES) \
```

Comment:
I looked in our code base but can't seem to find any `#ifdef` where we have `||` at the beginning of the line. I think it should go at the end of the previous line.

Reply:
Done